client/my-sites/patterns/index.node.tsx

```diff
@@ -41,6 +42,7 @@ function renderPatterns( context: RouterContext, next: RouterNext ) {
 				<PatternLibrary
 					categoryGallery={ CategoryGalleryServer }
 					patternGallery={ PatternGalleryServer }
+					readymadeTemplates={ ReadymadeTemplatesServer }
```

Comment:
Considering that `ReadymadeTemplatesServer` and `ReadymadeTemplatesSection` are the exact the same component, I wonder if we can remove `ReadymadeTemplatesServer` and just use `ReadymadeTemplatesSection` here.